web/router.ex

```diff
@@ -9,14 +9,29 @@ defmodule Pairmotron.Router do
     plug :put_secure_browser_headers
   end
 
+  pipeline :authenticate do
+    plug Pairmotron.Plug.Authenticate
+  end
+
   pipeline :api do
     plug :accepts, ["json"]
   end
 
   scope "/", Pairmotron do
-    pipe_through :browser # Use the default browser stack
+    pipe_through :browser
+
+    get "/registration", RegistrationController, :new
+    post "/registration", RegistrationController, :create
+
```

Comment:
could we replace this with `resources "registration", RegistrationController, only: [:new, :create]` ?